### DISCUSSION TITLE
Unable to Pull Auction Draft Info

### DISCUSSION THREAD

**Discussion by Nielwulf** [cwendt94/espn-api#220] [ESPN API] [answered]
When I try to gather league draft info, 99% of the list of picks only contains the Team that made the pick, not the player, with the 1% actually containing the player name. Anyone else seeing this issue? I would like to do some rank/draft pick charts comparisons.

**Comment by cwendt94** [comment, marked as answer]
Hi @Nielwulf, is this for previous seasons? The draft data is pretty simple and looks up `playerIds` in a player map. You can see all of the players doing `league.player_map` in python CLI after initializing the league.

**Reply by Nielwulf** [reply]
I'm trying to pull data from last year's draft. 

However today it is working now. Yesterday it looked like, 

Pick(, Team(team))

But now today it looks like it is working as intended.

Pick(Ryquell Armstead, Team(team))

The only thing that I could think of making a difference is that yesterday afternoon the draft settings were determined for this year, and I'm wondering if that made the difference.

**Reply by Nielwulf** [reply]
What was also weird was that it would show, Davante Adams, Cam Akers, and Henry Ruggs III. no other player names would come through.

Another thing was, if I were to print out the details of the pick, it would have them all except the player's name. It was just super strange.

**Reply by cwendt94** [reply]
Yeah that does sound really strange. It could have been an issue with ESPN's API, but I am glad it is working now!

**Reply by TheWorldIMN** [reply]
Hello Nielwulf, I am wondering if you were able to get the auction price per player as well. I have a keeper league and would like to use the API to bring in keeper value of a player, and getting the auction prices would be a great place to start on my task.

**Reply by Nielwulf** [reply]
Sorry my answer is late as you probably already drafted but it looked to work itself out when the ESPN API started to give me actual info.

In my league we also keep track of waiver wire costs to grab a player in our keeper values. So I still have to do some form of direct API calls to get what we need to get accurate keeper values.

**Reply by TheWorldIMN** [reply]
It's no problem, Niel. Thanks for reaching out. I'm going to play around with the API and see what I can find in there. Our keeper league has an auction draft and an FAAB waiver process, so keeper prices are higher of the auction price or the pick up price, plus a tax based on number of keepers. So yeah... Python and this API can definitely help me lol.
